web/satellite/src/components/buckets/BucketItem.vue

```diff
@@ -3,7 +3,7 @@
 
 <template>
     <div class="container">
-        <div class="container__item">{{ name }}</div>
+        <div class="container__item" :title="itemData.bucketName">{{ name }}</div>
```

Comment:
same here, would it be easier to read and understand if we will name it ```bucket```, not itemData?